sycl/doc/extensions/experimental/sycl_ext_oneapi_memory_export.asciidoc

```diff
@@ -0,0 +1,354 @@
+    size_t alignment, size_t size,
+    external_mem_handle_type externalMemHandleType,
+    const sycl::device &syclDevice, const sycl::context &syclContext,
+    const property_list& propList = {});
+
+void *alloc_exportable_device_mem(
+    size_t alignment, size_t size,
+    external_mem_handle_type externalMemHandleType,
+    const sycl::queue &syclQueue,
+    const property_list& propList = {});
+}
+```
+
+The `alloc_exportable_device_mem` function allocates memory on the device marked
+as having the ability to later export that memory to an external memory resource
+handle.
+
+Memory allocated through this function must only be freed using
+`free_exportable_mem`. Using `sycl::free` to deallocate memory allocated with
+this function results in undefined behavior.
+
+With the exception of the `sycl::free` function from the core SYCL
+specification, pointers to memory allocated through this function may be passed
+to any core SYCL specification API accepting device USM memory pointers.
+
+Memory allocated through this function is only available on device.
+
+Memory allocated through this function has a linear memory layout on the device 
+(which is the same as memory allocated by other USM allocation functions like 
+`sycl::malloc_device`).
+
+Zero or more properties can be passed in the `propList` parameter via an
+instance of `sycl::property_list`. Currently, this extension does not define
+any properties that can be used with this function, so the `propList` parameter
+is ignored and reserved for future use.
+
+Only two values of `externalMemHandleType` are supported by this extension:
+
+- `external_mem_handle_type::opaque_fd` is supported when the host is a Posix
+  compliant operating system.
+
+- `external_mem_handle_type::win32_nt_handle`` is supported when the host is
+  Windows.
+
+No other values are supported. This function will throw a `sycl::exception` with
+the `errc::feature_not_supported` code if an unsupported value is passed.
+
+This function will throw a `sycl::exception` with `errc::feature_not_supported`
+if the device `syclDevice` does not have
+`aspect::ext_oneapi_exportable_device_mem`.
+
+This function will throw a `sycl::exception` with the `errc::runtime` code if
+any error occurs while allocating the memory.
+
+```c++
+
+namespace sycl::ext::oneapi::experimental {
+
+template <external_mem_handle_type ExternalMemHandleType>
+__return_type__
+export_device_mem_handle(void *deviceMemory, const sycl::device &syclDevice,
+                         const sycl::context &syclContext);
+
+template <external_mem_handle_type ExternalMemHandleType>
+__return_type__
+export_device_mem_handle(void *deviceMemory, const sycl::queue &syclQueue);
+
+}
+```
+
+Constraints: `ExternalMemHandleType` is either
+`external_mem_handle_type::opaque_fd` or
+`external_mem_handle_type::win32_nt_handle`.
+
+When `ExternalMemHandleType` is `external_mem_handle_type::opaque_fd`, the
+`__return_type__` is `int`.
```

Comment:
```suggestion
`+__return_type__+` is `int`.
```

You need this to render the underbars verbatim.  Otherwise, they are interpreted as italics formatting.